CHANGELOG.md

```diff
@@ -3,7 +3,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## [2.2.0] - Unreleased
+## [3.0.0] - 2020-03-11
+## Changed
+- Add support for ruby 2.7
+- Remove support for ruby < 2.3
+- Change required version for gem rack to fit for all supported ruby versions
```

Comment:
Actually I think you could tell that it was changed the required version of the gem `heartcheck`, because it is the only change in gemspec file.

WDYT?